src/uptane/secondaryconfig.h

```diff
@@ -22,6 +23,7 @@ class SecondaryConfig {
   bool partial_verifying;
   std::string ecu_private_key;
   std::string ecu_public_key;
+  KeyType key_type{kRSA2048};
```

Comment:
This is basically hardcoding all secondaries to use RSA 2048. That's not terrible, but we should maybe instead add the ability to set this via `Config::readSecondaryConfigs` and change our examples and test data to try it out.